routers/install/install.go

```diff
@@ -558,6 +558,8 @@ func SubmitInstall(ctx *context.Context) {
 			}
 			log.Info("Admin account already exist")
 			u, _ = user_model.GetUserByName(ctx, u.Name)
+		} else {
+			ctx.Data["IsAccountCreated"] = true
```

Comment:
It seems an abuse to assign "post-install" related data in the "SubmitInstall" handler. "post-install" page has its own handler.

And "post-install" could be refreshed, then you lose the data here.